### DISCUSSION TITLE
T-Display-S3 AMOLED: Screen Remains Black Despite Functionality in Console and Multiple Flash Attempts

### DISCUSSION THREAD

**Discussion by DrBlender** [BitMaker-hub/NerdMiner_v2#464] [Q&A] [answered]
Hey, I own the T-Display-S3 AMOLED. It works perfectly, as I can see in the console. Unfortunately, the screen remains black. I have already tried flashing it myself, as well as using websites that offer it, but the screen still stays black. How can I fix this issue?

**Comment by jsy-rg1570** [comment]
> Hey, I own the T-Display-S3 AMOLED. It works perfectly, as I can see in the console. Unfortunately, the screen remains black. I have already tried flashing it myself, as well as using websites that offer it, but the screen still stays black. How can I fix this issue?

Dear, what firmware have you flashed ?

**Comment by DrBlender** [comment]
I'm sorry, I forgot to mention that. I am using the LILYGO TDisplay-AMOLED versions with firmware 1.6.3.

**Reply by jsy-rg1570** [reply]
May you send a picture of your board please ?

**Reply by DrBlender** [reply]
![WhatsApp Image 2024-08-22 at 23 12 38](https://github.com/user-attachments/assets/d5582435-d5ff-42aa-bc4d-23d50a365739)

**Reply by jsy-rg1570** [reply]
> ![WhatsApp Image 2024-08-22 at 23 12 38](https://github.com/user-attachments/assets/d5582435-d5ff-42aa-bc4d-23d50a365739)
> 
> 

Can you flip it please ?

**Reply by DrBlender** [reply]
![WhatsApp Image 2024-08-22 at 23 21 35](https://github.com/user-attachments/assets/30993bdf-6ae6-4ca1-9ace-68e123172a60)

**Reply by jsy-rg1570** [reply, marked as answer]
It should be the Lilygo AMOLED V2 which is very similar to AMOLED touch, so you could try AMOLED touch firmware.

**Reply by jsy-rg1570** [reply]
![image](https://github.com/user-attachments/assets/ebccc1c4-8dae-4e57-82f3-c7a8fb171340)

**Comment by DrBlender** [comment]
Thank you, now it works fine!

**Comment by tehbra1n** [comment]
I ran into the same issue with a Lilygo AMOLED V2 firmware and flashed the prerelease firmware for the touch version and it works fine.
